app/models/setting/ansible.rb

```diff
@@ -0,0 +1,19 @@
+class Setting::Ansible < ::Setting
+  def self.load_defaults
+    return unless ActiveRecord::Base.connection.table_exists?('settings')
```

Comment:
Not sure, this is from the instructions in http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin